### PR TITLE
Remove unused and commented code

### PR DESCRIPTION
**Pull request by MaxGhenis**
Minor cleanup only. Python 2 is deprecated, so the `__future__` import is no longer needed, and TODO functions should be in issues or PRs.

### CODE REVIEW

**Comment by toobaz** [comment]
Makes sense, happy to merge if you can rebase.

**Comment by MaxGhenis** [comment]
Rebased

File: stargazer/stargazer.py [hunk 1]
```diff
@@ -696,13 +695,4 @@ def generate_additional_notes(self):
...
-
-
-    # Begin Markdown render functions
-    # def render_markdown(self):
-    #     print("sorry haven't made this yet :/")
-
-    # Begin ASCII render functions
-    # def render_ascii(self):
-    #     print("sorry haven't made this yet :/")
+        return notes_text
```
Comment (toobaz) [inline stargazer/stargazer.py:698]: This wasn't garbage ;-) https://stackoverflow.com/a/729795/2858145

**Comment by toobaz** [comment]
Thanks @MaxGhenis !

**Pull request merged**